brain-marks/Categories/Views/CategoryList.swift

```diff
@@ -82,9 +78,12 @@ struct CategoryList: View {
     
     var categories: some View {
         List {
-            ForEach(viewModel.categories) { category in
+            ForEach(viewModel.categories, id: \.self) { category in
                 NavigationLink(destination: TweetList(category: category)) {
-                    CategoryRow(category: category)
+                    CategoryRow(
```

Comment:
This needed to change to take in both the name and image, since those change but the ID doesn't. Since the ID wasn't changing this wasn't updating.